app-config.yaml

```diff
@@ -56,6 +56,11 @@ proxy:
       # Authorization: 'Bearer <YOUR TOKEN>'
     changeOrigin: true
     secure: true
+  '/snyk':
+    target: https://snyk.io/api/v1
+    headers:
+      User-Agent: tech-services/backstage-plugin/1.0
+      Authorization: token ${SNYK_TOKEN}
```

Comment:
Is there any way you could add this information to the [getting-started documentation](https://github.com/janus-idp/backstage-showcase/blob/main/docs/getting-started.md). We have other teams interested in using our image and having the configurations present in the doc will help them get setup.